File: leetcode-submissions/386_Lexicographical_Numbers.cpp
class Solution {
public:
static bool cmp(const int & a,const int & b){
return to_string(a)<to_string(b);
}
    vector<int> lexicalOrder(int n) {
        vector<int>ans;
        for(int i=1;i<=n;i++){
            ans.push_back(i);
        }
        sort(ans.begin(),ans.end(),cmp);
        return ans;
    }
};

File: leetcode-submissions/386_Lexicographical_Numbers.ts
function lexicalOrder(n: number): number[] {
    const ans:number[]=[];
    for(let i=1;i<=n;i++){
        ans.push(i);

    }
    ans.sort((a,b)=>(a.toString()).localeCompare(b.toString()));
    return ans;
};